app/fissile.go

```diff
@@ -351,14 +334,15 @@ func (f *Fissile) GeneratePackagesRoleImage(repository string, roleManifest *mod
 		docker.ColoredBuildStringFunc(packagesLayerImageName),
 	)
 
-	err = dockerManager.BuildImageFromStream(tarStream, packagesLayerImageName, stdoutWriter)
+	tarPopulator := packagesImageBuilder.PopulateTarStream(roleManifest, lightManifestPath, darkManifestPath, force)
```

Comment:
Yep, I didn't like that name. So I went for the [xkcd 386](https://www.xkcd.com/386/) way of getting a better one :grinning: